supabase/migrations/20240130152000_create_model.sql

```diff
@@ -35,5 +35,26 @@ CREATE TABLE IF NOT EXISTS games (
     second_player_id INT REFERENCES users(id) ON DELETE CASCADE,
     creation_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
     last_update_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    game_state VARCHAR(255) NOT NULL
-);
\ No newline at end of file
+    game_state JSONB NOT NULL
+);
+
+CREATE INDEX idx_games_winner ON games ((game_state->'victoryState'->>'player'));
+
+CREATE VIEW games_view AS
+    SELECT 
```

Comment:
The idea of migrations is to use several files, to know which migration was already applied and which was not.
=> You should create a new migration file each time you need to update the db schema